starlite/app.py

```diff
@@ -401,22 +402,22 @@ def __init__(
         for plugin in (p for p in config.plugins if isinstance(p, InitPluginProtocol)):
             plugin.on_app_init(app=self)
 
-        for route_handler in config.route_handlers:
+        for route_handler in config.route_handlers:  # type: ignore
             self.register(route_handler)
 
         if self.debug and isinstance(self.logging_config, LoggingConfig):
             self.logging_config.loggers["starlite"]["level"] = "DEBUG"
 
         if self.logging_config:
-            self.get_logger = self.logging_config.configure()
+            self.get_logger = self.logging_config.configure()  # type: ignore
```

Comment:
I think the ignore is necessary here as we allow the logging config to be both `None` or `Empty` but `Empty` doesn't evaluate falsey, so at this point the config object could be `Empty` or a config object. It that is the case, the type error is correct and we shouldn't be ignoring it.